dwarf-op.lua

```diff
@@ -1076,15 +915,19 @@ function CheckWorker(dwf, option)
             local select_type = list[1]
             if GetChar(select_type,1) == 'p' or select_type == 'name' or select_type == 'names' then
                 includeProtectedDwfs = true
-                select_type = string.sub(select_type,2)
+                if GetChar(select_type,1) == 'p' then
```

Comment:
`GetChar` is fine since it's used in other places, but be aware that dfhack.lua also adds `string:startswith()`, so when getting the first character and comparing it you could write it as `if select_type:startswith('p')` instead of `if GetChar(select_type, 1) == 'p'`